cloud-sql/mysql/client-side-encryption/README.md

```diff
@@ -0,0 +1,43 @@
+# Encrypting fields in Cloud SQL - MySQL with Tink
+
+## Before you begin
+
+1. If you haven't already, set up a Java Development Environment (including google-cloud-sdk and
+maven utilities) by following the [java setup guide](https://cloud.google.com/java/docs/setup) and
+[create a project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project).
+
+1. If you haven't already, set up a Python Development Environment by following the [python setup guide](https://cloud.google.com/python/setup) and 
+[create a project](https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project).
```

Comment:
👀

Reply:
Oh no.. my copy-pasting has been exposed! Fixed in latest commit.